projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -64,6 +64,23 @@ export class TuiTiptapEditorService extends AbstractTuiEditor {
         });
     }
 
+    @tuiPure
```

Comment:
Are we sure there is no case when content of the editor was changed some other way and calling `setValue` with the same string as we did before would not be a valid situation to process? Seems like we are only saving up an extra call to `getHTML()` at the cost of uncertainty.

Reply:
fixed